src/main/java/com/training/springpractice/models/Catalog.java

```diff
@@ -0,0 +1,75 @@
+package com.training.springpractice.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Entity(name = "movie_catalog")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+public class Catalog {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "movie_id", referencedColumnName = "id")
+    private Movie movie;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "price_id", referencedColumnName = "id")
+    private Price price;
+
+    @NotNull
+    @Min(0)
```

Comment:
I'm not sure this annotattions will work as you expected, maybe you need to review it, some annotations are used for bean validations and others are used as Hibernate or JPA annotations, in this last scenario those are used to enforce the database defintion/structure.  Please review if this annotation will work as you expect.